lib/phoenix-campaign.js

```diff
@@ -13,6 +13,9 @@ class PhoenixCampaign {
     this.status = data.status;
     this.uri = data.uri;
     this.type = data.type;
+    this.currentRun = {
+      en: data.campaign_runs.current.en.id,
```

Comment:
@sergii-tkachenko -- do you suggest just setting the `en` id as `currentRun` ?

e.g. `campaign.currentRun == 6270`